src/OptionsPopup.qml

```diff
@@ -404,15 +404,14 @@ Window {
                     // textFormat: Text.PlainText
                     Layout.leftMargin: 40
                 }
-                TextArea {
+                TextField {
```

Comment:
Does the switch from `TextArea` to `TextField` mean that advanced users will no longer be able to specify _multiple_ public SSH keys in this field?

Reply:
Yes - this breaks that case in favour of introducing a real-time validator. TextArea has no such equivalent mechanism.

Reply:
Do keep in mind that support for multiple keys was an explicit user request, so this is likely to lead to complaints. It was once a textfield and people opened issues about that: https://github.com/raspberrypi/rpi-imager/issues/316